src/test/java/tests/MoodPandaTest.java

```diff
@@ -0,0 +1,24 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class MoodPandaTest extends BaseTest {
+
+    @Test(invocationCount = 2)
+    public void loginTest() {
+        loginPage.openPage()
+                .login(EMAIL, PASSWORD)
+                .waitForPageOpen()
+                .clickOnUpdateMoodButton()
+                .waitForPageOpen();
```

Comment:
make method "waitForPageOpen" to return objects and you sctructure will be like:
        loginPage.openPage()
                .login(EMAIL, PASSWORD)
                .waitForPageOpen()
                .clickOnUpdateMoodButton()
                .waitForPageOpen();
                .updateMood(8, "HELLO!")
                .clickUpdateMoodToButton()
                .waitForPageOpen();
                .clickDoneButton();
                .waitForPageOpen();

Reply:
done